pkg/cloudprovider/aws/fleet/packing/instancetype.go

```diff
@@ -0,0 +1,96 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package packing
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+// TODO get this information from node-selector
+var (
+	nodePools = []*nodeCapacity{
```

Comment:
I'd avoid calling this node pools since GKE has a concept of node pool and this might be misleading. 

capacityTypes := []*capacityType

Reply:
Oh I see this object is mutable (reserve), so maybe we can just call it capacity? capacity.reserve()